spec/lib/scrabble_spec.rb

```diff
@@ -0,0 +1,99 @@
+describe Scrabble::Scrabble do
+
+  describe '.convert_input(word)' do
+    it 'converts word into an array of lowercase letters' do
+      expect(Scrabble::Scrabble.convert_input("CaTs")).to eq(["c","a","t","s"])
+    end
+  end
+
+  describe ".acceptable_input_check" do
+    before do
+      $stdin = StringIO.new("cat\n")
+    end
+
+    after do
+      $stdin = STDIN
+    end
+
+    it "does not accept nil" do
```

Comment:
Way to think about testing input.  This helps a newcomer to your code understand how the methods should behave.